__tests__/user/user.query.test.js

```diff
@@ -4,25 +4,64 @@ const client = require('../../utils/apollo-client');
 
 require('dotenv').config();
 
-let token
+let token, userId
 
 describe('queries', () => {
-    beforeEach(async () => {
-        const res = await client.mutate({
+    beforeAll(async () => {
+        const register = await client.mutate({
+            mutation: gql`
+                mutation {       
+                    registerUser(user: {
+                        firstName: "Test" 
+                        lastName: "User"
+                        email: "test.email@gmail.com"
+                        password: "12345678Te"
+                    }) {
+                        _id
+                        firstName
+                        lastName
+                        email
+                        role
+                        registrationDate
+                        credentials {
+                            tokenPass
+                        } 
+                    }
+                }
+            `
+        })
+
+        userId = register.data.registerUser._id
+
+        const auth = await client.mutate({
             mutation: gql`
                 mutation {       
                     loginUser(user: {
-                        email: "tacjka334@gmail.com"
-                        password: "12345678Pt"
+                        email: "test.email@gmail.com"
+                        password: "12345678Te"
                     }) {
                         token
                     }
                 }
             `
         })
-        token = res.data.loginUser.token
+        token = auth.data.loginUser.token
```

Comment:
Please, use `authRes` to emphasize that it is the response because `auth` is too vague name

Reply:
fixed